### DISCUSSION TITLE
Grist Behind Proxy

### DISCUSSION THREAD

**Discussion by jsapede** [community-scripts/ProxmoxVE#1132] [Q&A] [answered]
looks like grist only works from inside LAN, once proxied get errors : 

![image](https://github.com/user-attachments/assets/7ddeb923-b1cd-4d81-89d5-d5e200f3687b)

didnt found in the install where is the config to find if a domain url must be sepcified

**Comment by jsapede** [comment, marked as answer]
Ok finally found it exploring the script, its in /opt/grist/.env

**Reply by gh2023-aesun** [reply]
I have the same issue with the Grist LXC Helper Script install.  I've messed around with changing the GRIST_HOST variable, adding GRIST_DOMAIN variable, all in vain.

I have a docker based Grist implementation as well, which works with Nginx Proxy Manager and the following settings:
```
Cache Assets - Off
Block Common Exploits - Off
Websockets Support - On
ForceSSL - On
HTTP/2 Support - On
HSTS Enabled - On
HSTS Subdomains - On
```

The same settings do not work with the Grist LXC Helper Script install with default settings.

What changes did you make to /opt/grist/.env to get this to work?

**Reply by gh2023-aesun** [reply]
Nevermind, I solved my issue by adding the following variable to  /opt/grist/.env : 
`APP_HOME_URL=https:// ...`
